doc/rangeproofs.md

```diff
@@ -0,0 +1,132 @@
+# Range Proofs - A Primer
+
+This document is intended to give the reader a high level conceptual overview of what range proofs are and how they're contructed. Understanding range proofs is not 100% necessary in order to understand Mimblewimble, however it is necessary to understand that the blinding factor used to create an output on the block chain must be known in order to create a valid range proof. An explantion of why this is the case follows through an elementary example of the creation of a range proof. 
```

Comment:
Maybe also reiterate the main reason for a range proof to exist at all: proving with zero-knowledge the range of the value (otherwise there are simpler ways to prove ownership of the private key).